PowerEditor/src/dpiManagerV2.cpp

```diff
@@ -286,4 +286,10 @@ void DPIManagerV2::loadIcon(HINSTANCE hinst, const wchar_t* pszName, int cx, int
 	{
 		*phico = static_cast<HICON>(::LoadImage(hinst, pszName, IMAGE_ICON, cx, cy, fuLoad));
 	}
+
+	// in case of LoadIconWithScaleDown is not compatible on Windows version: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/15313
+	if (!*phico)
```

Comment:
this can be added to previous check
`if ((::LoadIconWithScaleDown(hinst, pszName, cx, cy, phico) != S_OK) || (*phico == nullptr))`

[LoadImage](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-loadimagew) is supported from Windows 2000 and even MS recommend to use it instead of `LoadIcon`